docs/src/installing-deis/installing-the-deis-platform.md

```diff
@@ -35,78 +35,156 @@ $ kubectl get pods --namespace=deis
 
 Once you see all of the pods ready, your Deis platform is running on a cluster!
 
-## Mapping a Default Domain
+## Routing Traffic
 
-Deis will route traffic from Kubernetes nodes to Deis components, but to
-do so it needs a domain. By default, Deis is configured to use the
-domain `example.com`. For basic Deis testing, simply add an entry to
-your host machine's `/etc/hosts` file.
+The Deis router component will use an incoming request's HTTP `Host`
+header to direct traffic to Deis components and to applications deployed
+via Deis. It is therefore important that the router be configured
+properly and that DNS and/or your local `/etc/hosts` file are as well.
+
+### Configuring the Platform Domain
+
+The Deis router requires a "platform domain" be set.  Deis components
+that respond to HTTP requests and applications deployed via Deis will
+all live on subdomains of the platform domain.  When installing Deis
+via helm, the platform domain is initially set to `example.com`.  This
+can be changed by editing Deis' helm chart prior to installation.
+Alternatively, the change can be made after Deis has been installed
+by editing the router's replication controller like so:
+
+```
+$ kubectl edit rc deis-router --namespace=deis
+```
+
+Find `example.com` in the manifest, change it to a domain of your
+choosing, and save changes.  Kubernetes will apply the changes and
+the router will dynamically reconfigure itself accordingly.
+
+### Getting Traffic to the Cluster
+
+With the platform domain configured, the only remaining concern
+is to ensure traffic for that domain reaches the cluster.  There
+are a few ways of doing this, and which you use may depend on the
+capabilities of your infrastructure and your Kubernetes cluster.
+
+In general, the goal is to connect traffic bound for
+`*.the-domain-you.picked` to your Kubernetes cluster.
+
+#### Using an Automatically Provisioned Load Balancer
+
+If your Kubernetes cluster and the underlying infrastructure
+support it, the Deis router's service (which is of `type:
+LoadBalancer`) will automatically provision an "external"
+(external to the Kubernetes cluster) load balancer.  On AWS,
+for instance, this takes the form of an ELB, while on Google
+Container Engine, it takes the form of a routing rule.
+
+If it is supported, you will be able to find the DNS name or
+IP(s) of the external load balancer like so:
+
+```
+$ kubectl describe service deis-router --namespace=deis
+```
+
+Examine the `LoadBalancer Ingress` field to find the DNS
+name or IP(s).  The former can be added to DNS as a CNAME,
+while the latter can be added as an A record.
+
+#### Creating Your Own Load Balancer
+
+On Kubernetes clusters and underlying infrastructure that
+do not support the automated provisioning of an external
+load balancer for the Deis router, you may provision your
+own and add _all_ worker nodes of the Kubernetes cluster to
```

Comment:
Is minion a more proper term to use than worker?  I feel I see both terms uses interchangeably.  I do think it's useful if we all stick to a common lexicon, so if minion is prefered, I will change this.